### ISSUE TITLE
"Accessibility & Nondiscrimination" Heading label is incorrect.

### ISSUE THREAD

**Issue by bhattashubham** [veshu/accessibility-bughunt-bluesource#61]
## Steps to reproduce the behavior
1. Go to Wellmark App.
2. Go to login page 
3. scroll down to bottom and click on Accessibility & Nondiscrimination
4. Listen the heading label 

## Expected behavior:
Accessibility & Nondiscrimination heading label should be heading 1
Device: Laptop

Operation System: Windows 10

Browser: Microsoft Edge 44.17763.771.0

WCAG Category: 2. Operable

WCAG Level AA

WCAG Success Criterion 2.4.6 Headings and Labels

Tools used: Narrator 

Video:
https://drive.google.com/open?id=1kgBAY5VUOktralN00P-Drd4rqra67D2o
